### PR TITLE
[FLINK-10736][E2E tests] Switch to new IT_CASE_S3* and static/temp s3 content

### PR DESCRIPTION
**Pull request by azagrebin**
## What is the purpose of the change

Remove s3 delete from s3 shaded e2e tests and use already uploaded, never  #deleted

## Verifying this change

run s3 shaded e2e tests

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no, just tests)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

File: flink-end-to-end-tests/test-scripts/test_shaded_hadoop_s3a.sh [hunk 1]
```diff
@@ -22,15 +22,11 @@
...
-s3_put $TEST_INFRA_DIR/test-data/words $ARTIFACTS_AWS_BUCKET flink-end-to-end-test-shaded-s3a
-# make sure we delete the file at the end
-function shaded_s3a_cleanup {
-  s3_delete $ARTIFACTS_AWS_BUCKET flink-end-to-end-test-shaded-s3a
-}
-trap shaded_s3a_cleanup EXIT
-
 start_cluster
 
+bucket=flink-e2e-tests
```
Comment (dawidwys) [inline flink-end-to-end-tests/test-scripts/test_shaded_hadoop_s3a.sh]: Could we move the definition of `bucket` to `common_s3.sh`? Also I think we should use uppercase here to follow the convention of constants like: `TEST_INFRA_DIR`, `ARTIFACTS_AWS_BUCKET` etc.

File: flink-end-to-end-tests/test-scripts/test_shaded_presto_s3.sh [hunk 2]
```diff
@@ -22,15 +22,11 @@
 source "$(dirname "$0")"/common.sh
 source "$(dirname "$0")"/common_s3.sh
 
-s3_put $TEST_INFRA_DIR/test-data/words $ARTIFACTS_AWS_BUCKET flink-end-to-end-test-shaded-presto-s3
```
Comment (zentol) [inline flink-end-to-end-tests/test-scripts/test_shaded_presto_s3.sh]: Could we still do the put (to guard against accidental deletes etc.) without affecting test stability?

Comment (azagrebin) [reply flink-end-to-end-tests/test-scripts/test_shaded_presto_s3.sh]: I think this is good point. We can just remove trap with deletion of the file atm.

**Comment by StephanEwen** [comment]
Fixed in aabbd6576c7608cf78f2da5b7c5170135ad2db54

**Pull request closed without merging**